conans/client/graph/sbom.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from conan.internal.cache.home_paths import HomePaths
+
+_default_spdx_json = """
```

Comment:
Put this under a ``spdx.py`` file or something like that.

Reply:
Quick question: How would then a user "override" the generated sbom if we're only creating/looking for this file. If they decide to use cyclone for example, having the file still named spdx would be a bit suprising

Reply:
The ``spdx.py`` and other potential sbom and manifests utilities should be imported only from the ``sbom.py`` (or whatever name we give it) plugin implementation, and maybe from recipes if we consider it. Maybe we want to put them in a ``conan.tools.sbom`` or ``conan.tools.xxx`` package?